server/plugin.go

```diff
@@ -169,9 +169,14 @@ func (p *Plugin) OnConfigurationChange() (err error) {
 	p.tracker.UpdateTracker(updatedTracker)
 
 	mm := pluginapi.NewClient(p.API, p.Driver)
-	cm := mm.Configuration.GetPluginConfig()
+	var sc config.StoredConfig
+	err := mm.Configuration.LoadPluginConfiguration(&sc)
+	if err != nil {
+		p.API.LogInfo("failed to load updated configuration", "error", err.Error())
```

Comment:
Should this be warn instead?

```suggestion
		p.API.LogWarn("failed to load updated configuration", "error", err.Error())
```